packages/models-library/src/models_library/volumes.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+from enum import auto
+
+import arrow
+from pydantic import BaseModel, Field
+
+from .utils.enums import StrAutoEnum
+
+
+class VolumeCategory(StrAutoEnum):
+    """
+    These uniquely identify volumes which are mounted by
+    the dynamic-sidecar and user services.
+
+    This is primarily used to keep track of the status of
+    each individual volume on the volumes.
+
```

Comment:
you might want to rename that module. or maybe put it in a folder *dy-sidecar* or so? volumes is a very generic name